bin/install-theos

```diff
@@ -0,0 +1,598 @@
+	if [[ -d "$THEOS" && "$(ls -A "$THEOS")" ]]; then
+		update "Theos appears to already be installed. Checking for updates..."
+		$THEOS/bin/update-theos
+	else
+		update "Theos does not appear to be installed. Cloning now..."
+		git clone --recursive https://github.com/theos/theos.git $THEOS && update "Git clone of Theos was successful!" || (error "Theos git clone command seems to have encountered an error. Please see the log above." && exit 9)
+	fi
+
+	# Get patched sdks
+	update "Checking for patched sdks..."
+	if [[ -d "$THEOS/sdks/" && "$(ls -A "$THEOS/sdks/" | grep sdk)" ]]; then
+		update "SDKs appear to already be installed."
+	else
+		update "SDKS do not appear to be installed. Installing now..."
+		curl -LO https://github.com/theos/sdks/archive/master.zip
+		TMP=$(mktemp -d)
+		unzip master.zip -d $TMP
+		mv $TMP/sdks-master/*.sdk $THEOS/sdks
+		rm -r master.zip $TMP
+
+		if [[ -d "$THEOS/sdks/" && "$(ls -A "$THEOS/sdks/" | grep sdk)" ]]; then
+			update "SDKs successfully installed!"
+		else
+			error "Something appears to have gone wrong. Please try again."
+			exit 10
+		fi
+	fi
+}
+
+darwin-mobile() {
+	common "Platform: Darwin (Mobile)"
+
+	# Check for sudo (not installed by default on some jbs)
+	if ! command -v sudo &> /dev/null; then
+		error "Please install 'sudo' in your package manager before proceeding with the installation."
+		exit 3
+	fi
+
+	# Determine iOS version
+	OLD=0
+	KERNEL_VER=$(sysctl kern.osrelease | sed 's/[^0-9]*//g')
+	if [[ $KERNEL_VER < 1800 ]]; then
+		OLD=1
+	fi
+
+	# Dependencies
+	update "Preparing to install dependencies. Please enter your password if prompted:"
+	if [[ $OLD == 1 ]]; then
+		read -p "Do you have 'https://coolstar.org/publicrepo/', 'https://repo.bingner.com/', and 'http://apt.thebigboss.org/repofiles/cydia/' installed in the sources of your jailbreak's primary package manager? [y/n]" ready
+		if [[ "${ready,,}" == "y" || "${ready,,}" == "yes" ]]; then
+			sudo apt install org.theos.dependencies -y --allow-unauthenticated --allow-downgrades && update "Dependencies have been successfully installed!" || (error "Dependency install command seems to have encountered an error. Please see the log above." && exit 5)
+		else
+			error "Please install the repos mentioned above before proceeding."
+			exit 4
+		fi
+	else
+		# Dependencies pkg is outdated, so need to cater install for Procursus vs Elucubratus
+		if [[ -f "/.procursus_strapped" ]]; then
+			read -p "Do you have 'https://apt.procursus.us' installed in the sources of your jailbreak's primary package manager? [y/n]" ready
+			if [[ "${ready,,}" == "y" || "${ready,,}" == "yes" ]]; then
+				sudo apt install ca-certificates clang coreutils curl dpkg git grep ldid make odcctools perl plutil rsync unzip xz-utils -y --allow-unauthenticated --allow-downgrades && update "Dependencies have been successfully installed!" || (error "Dependency install command seems to have encountered an error. Please see the log above." && exit 5)
```

Comment:
Pro has an org.theos.dependencies package so we can install that there.

Reply:
Yup. For some reason Proc isn't detected as signed immediately after jailbreaking (can be resolved with an `apt update`), but since that's a rare case it shouldn't be a huge issue (or may be a fluke on my device).

Reply:
@L1ghtmann Probably best to apt update first anyway as you will want to install the latest versions of the dependencies, to get the best experience.